### PR TITLE
Making Commits for legacy entities

### PR DESCRIPTION
**Pull request by Dbarnes1**

### CODE REVIEW

**Comment by Dbarnes1** [comment]
@BobBooth @abouffard ignore the previous pull request that I closed, this is the one where I will be committing more of my changes. Just finished Air and Air Missile

**Comment by Dbarnes1** [comment]
@abouffard @BobBooth ready for review and merging.

File: instance/jmsml_D_Land_Unit.xml [hunk 1]
```diff
@@ -2504,36 +2548,60 @@
     </LegacySymbol>
     <LegacySymbol ID="FIRES_AIR_DEFENSE_COMPOSITE_SYM" Label="S*GPUCDC--*****" Remarks="Retired">
       <LegacyFunctionCode Name="2525C">UCDC--</LegacyFunctionCode>
-      <LegacyEntity ID="FIRES_AIR_DEFENSE_COMPOSITE_LE" Label="WAR.GRDTRK.UNT.CBT.ADF.CMPS" CloverGraphic="S-G-UCDC--_0.svg" RectangleGraphic="S-G-UCDC--_1.svg" SquareGraphic="S-G-UCDC--_2.svg" DiamondGraphic="S-G-UCDC--_3.svg" Icon="FULL_FRAME">
+      <LegacyEntity ID="FIRES_AIR_DEFENSE_COMPOSITE_LE" Label="WAR.GRDTRK.UNT.CBT.ADF.CMPS" Icon="Full_Frame" CloverGraphic="S-G-UCDC--_0.svg" RectangleGraphic="S-G-UCDC--_1.svg" SquareGraphic="S-G-UCDC--_2.svg" DiamondGraphic="S-G-UCDC--_3.svg" Icon="FULL_FRAME">
```
Comment (abouffard) [inline instance/jmsml_D_Land_Unit.xml]: @Dbarnes1 - Icon=FULL_FRAME appears twice in this line.  It only needs to be there once with the full caps.  Doesn't really matter where in the line.

File: instance/jmsml_D_Land_Unit.xml [hunk 2]
```diff
@@ -2504,36 +2548,60 @@
...
         <LegacyFunctionCode Name="2525C">UCDC--</LegacyFunctionCode>
       </LegacyEntity>
     </LegacySymbol>
     <LegacySymbol ID="FIRES_AIR_DEFENSE_SHORT_RANGE_MOD_SYM" Label="S*GPUCDS--*****" EntityID="FIRES" EntityTypeID="AIR_DEFENSE" ModifierTwoID="SHORT_RANGE_MOD">
       <LegacyFunctionCode Name="2525C">UCDS--</LegacyFunctionCode>
+	  <LegacyEntity ID="FIRES_AIR_DEFENSE_SHORT_RANGE_MOD_LE" Label="WAR.GRDTRK.UNT.CBT.ADF.SHTR" Icon="Full_Frame" CloverGraphic="S-G-UCDS--_0.svg" RectangleGraphic="S-G-UCDS--_1.svg" SquareGraphic="S-G-UCDS--_2.svg" DiamondGraphic="S-G-UCDS--_3.svg">
```
Comment (abouffard) [inline instance/jmsml_D_Land_Unit.xml]: @Dbarnes1 - All FULL_FRAME's should be full caps.

**Comment by Dbarnes1** [comment]
@abouffard fixed the full frame issue, you can go ahead and merge once verified.

File: instance/jmsml_D_Activity.xml [hunk 3]
```diff
@@ -1132,6 +1132,9 @@
     </LegacySymbol>
     <LegacySymbol ID="INCIDENT_CRIMINAL_ACTIVITY_INCIDENT_CIVIL_RIOTING_SYM" Label="E*IPAC----*****" EntityID="INCIDENT" EntityTypeID="CRIMINAL_ACTIVITY_INCIDENT" EntitySubTypeID="CIVIL_RIOTING">
       <LegacyFunctionCode Name="2525C" DimensionOverride="I">AC----</LegacyFunctionCode>
+	  <LegacyEntity ID="INCIDENT_CRIMINAL_ACTIVITY_INCIDENT_CIVIL_RIOTING_LE" Label="EMS.INCDNT.CVDIS.CVRIOT" Graphic="E-I-AC----.svg">
```
Comment (abouffard) [inline instance/jmsml_D_Activity.xml]: @Dbarnes1 - general comment.  All LegacyEntity structures should be indented.  They should line up with the LegacyFunctionCode above.

File: instance/jmsml_D_Air.xml [hunk 4]
```diff
@@ -823,27 +868,42 @@
...
+	  <LegacyEntity ID="MILITARY_UAV_ES_MOD_LE" Label="WAR.AIRTRK.MIL.FIXD.DRN.RECON.ESM" Graphic="S-A-MFQRZ-.svg">
+		<LegacyFunctionCode Name="2525C">MFQRZ-</LegacyFunctionCode>
+	  </LegacyEntity>
     </LegacySymbol>
     <LegacySymbol ID="MILITARY_UAV_TANKER_MOD_SYM" Label="S*APMFQK--*****" EntityID="MILITARY" EntityTypeID="UAV" ModifierOneID="TANKER_MOD">
       <LegacyFunctionCode Name="2525C">MFQK--</LegacyFunctionCode>
     </LegacySymbol>
     <LegacySymbol ID="MILITARY_UAV_VSTOL_MOD_SYM" Label="S*APMFQL--*****" EntityID="MILITARY" EntityTypeID="UAV" ModifierOneID="VSTOL_MOD">
       <LegacyFunctionCode Name="2525C">MFQL--</LegacyFunctionCode>
+	  <LegacyEntity ID="MILITARY_UAV_VSTOL_MOD_SYM" Label="WAR.AIRTRK.MIL.FIXD.DRN.VSTOL" Graphic="S-A-MFQL--.svg">
```
Comment (abouffard) [inline instance/jmsml_D_Air.xml]: @Dbarnes1 - the ID is not unique.  SYM needs to be LE.

File: instance/jmsml_D_Air.xml [hunk 5]
```diff
@@ -874,6 +943,9 @@
     </LegacySymbol>
     <LegacySymbol ID="MILITARY_UAV_COMMS_MOD_SYM" Label="S*APMFQY--*****" EntityID="MILITARY" EntityTypeID="UAV" ModifierOneID="COMMS_MOD">
       <LegacyFunctionCode Name="2525C">MFQY--</LegacyFunctionCode>
+	  <LegacyEntity Name="MILITARY_UAV_COMMS_MOD_LE" Label="WAR.AIRTRK.MIL.FIXD.DRN.COMM" Graphic="S-A-MFQY--.svg">
```
Comment (abouffard) [inline instance/jmsml_D_Air.xml]: @Dbarnes1 - ID is missing.

File: instance/jmsml_D_Air.xml [hunk 6]
```diff
@@ -823,27 +868,42 @@
     </LegacySymbol>
     <LegacySymbol ID="MILITARY_UAV_ACP_MOD_SYM" Label="S*APMFQD--*****" EntityID="MILITARY" EntityTypeID="UAV" ModifierOneID="ACP_MOD">
       <LegacyFunctionCode Name="2525C">MFQD--</LegacyFunctionCode>
+	  <LegacyEntity ID="MILITARY_UAV_ACP_MOD_LE" Label="WAR.AIRTRK.MIL.FIXD.DRN.ABNCP" Graphic="S-A-MFQD--.svg">
+		<LegacyFunctionCode Name="2525C">MFQD--</LegacyFunctionCode>
+	  </LegacyEntity>
     </LegacySymbol>
     <LegacySymbol ID="MILITARY_UAV_AEW_MOD_SYM" Label="S*APMFQRW-*****" EntityID="MILITARY" EntityTypeID="UAV" ModifierOneID="AEW_MOD">
       <LegacyFunctionCode Name="2525C">MFQRW-</LegacyFunctionCode>
+	  <LegacyEntity ID="MILITARY_UAV_AEW_MOD_SYM" Label="WAR.AIRTRK.MIL.FIXD.DRN.RECON.ABNEW" Graphic="S-A-MFQRW-.svg">
```
Comment (abouffard) [inline instance/jmsml_D_Air.xml]: @Dbarnes1 - SYM should be LE.

File: instance/jmsml_D_Land_Installation.xml [hunk 7]
```diff
@@ -995,6 +995,9 @@
     </LegacySymbol>
     <LegacySymbol ID="INSTALLATION_CBRN_RADIOLOGICAL_1_MOD_SYM" Label="S*GPIPD---H****" EntityID="INSTALLATION" EntityTypeID="CBRN" ModifierOneID="RADIOLOGICAL_1_MOD">
       <LegacyFunctionCode Name="2525C" HQTFFDOverride="H">IPD---</LegacyFunctionCode>
+	  <LegacyEntity ID="INSTALLATION_CBRN_RADIOLOGICAL_1_MOD_SYM" Label="WAR.GRDTRK.INS.PF.DECON" Graphic="S-G-IPD---.svg">
```
Comment (abouffard) [inline instance/jmsml_D_Land_Installation.xml]: @Dbarnes1 - SYM should be LE.

File: instance/jmsml_D_Land_Unit.xml [hunk 8]
```diff
@@ -2691,12 +2796,18 @@
     </LegacySymbol>
     <LegacySymbol ID="PROTECTION_CHEMICAL_BIOLOGICAL_RADIOLOGICAL_NUCLEAR_DEFENSE_SMOKE_MOD_DECONTAMINATION_2_MOD_SYM" Label="S*GPUUACC-*****" EntityID="PROTECTION" EntityTypeID="CHEMICAL_BIOLOGICAL_RADIOLOGICAL_NUCLEAR_DEFENSE" ModifierOneID="SMOKE_MOD" ModifierTwoID="DECONTAMINATION_2_MOD">
       <LegacyFunctionCode Name="2525C">UUACC-</LegacyFunctionCode>
+	  <LegacyEntity ID="PROTECTION_CHEMICAL_BIOLOGICAL_RADIOLOGICAL_NUCLEAR_DEFENSE_SMOKE_MOD_DECONTAMINATION_2_MOD_LE" Label="WAR.GRDTRK.UNT.CS.CBRN.CML.SMKDEC" Graphic="S-G-UUACC-.svg"
```
Comment (abouffard) [inline instance/jmsml_D_Land_Unit.xml]: @Dbarnes1 - missing closing angle.

File: instance/jmsml_D_Land_Unit.xml [hunk 9]
```diff
@@ -2658,18 +2748,33 @@
...
     <LegacySymbol ID="FIRES_SURVEY_AIRBORNE_MOD_SYM" Label="S*GPUCFSA-*****" EntityID="FIRES" EntityTypeID="SURVEY" ModifierTwoID="AIRBORNE_MOD">
       <LegacyFunctionCode Name="2525C">UCFSA-</LegacyFunctionCode>
+	  <LegacyEntity ID="FIRES_SURVEY_AIRBORNE_MOD_LE" Label="WAR.GRDTRK.UNT.CBT.FLDART.ARTSVY.ABN" Graphic="S-G-UCFSA-.svg">
+		<LegacyFunctionCode Name="2525C">UCFSA-</LegacyFunctionCode>
+	  </LegacyEntity>
     </LegacySymbol>
     <LegacySymbol ID="FIRES_SURVEY_LIGHT_MOD_SYM" Label="S*GPUCFSL-*****" EntityID="FIRES" EntityTypeID="SURVEY" ModifierTwoID="LIGHT_MOD">
       <LegacyFunctionCode Name="2525C">UCFSL-</LegacyFunctionCode>
+	  <LegacyEntity ID="FIRES_SURVEY_LIGHT_MOD_LE" Label="WAR.GRDTRK.UNT.CBT.FLDART.ARTSVY.LIT" Graphic="S-G-UCFSL-.svg">
+		LegacyFunctionCode Name="2525C">UCFSL-</LegacyFunctionCode>
```
Comment (abouffard) [inline instance/jmsml_D_Land_Unit.xml]: @Dbarnes1 - ??? :-)

File: instance/jmsml_D_Land_Unit.xml [hunk 10]
```diff
@@ -2352,9 +2367,16 @@
...
       <LegacyFunctionCode Name="2525C">UCRVO-</LegacyFunctionCode>
       <LegacyFunctionCode Name="2525C" Description="Repeat">UCVRS-</LegacyFunctionCode>
+	  <LegacyEntity ID="MOVEMENT_AND_MANEUVER_ARMY_AVIATION_AVIATION_ROTARY_WING_AVIATION_RECONNAISSANCE_LE" Label="WAR.GRDTRK.UNT.CBT.AVN.ROT.SCUT" Icon="FULL_FRAME" CloverGraphic="S-G-UCVRS-_0.svg" RectangleGraphic="S-G-UCVRS-_1.svg" SquareGraphic="S-G-UCVRS-_2.svg" DiamondGraphic="S-G_UCVRS-_3.svg">
+		<LegacyFunctionCode Name="2525C">UCRVO-</LegacyFunctionCode>
+        <LegacyFunctionCode Name="2525C" Description="Repeat">UCVRS-</LegacyFunctionCode>
+	  </LegacyEntity>
     </LegacySymbol>
     <LegacySymbol ID="MOVEMENT_AND_MANEUVER_AVIATION_COMPOSITE_SYM" Label="S*GPUCVC--*****" EntityID="MOVEMENT_AND_MANEUVER" EntityTypeID="AVIATION_COMPOSITE">
       <LegacyFunctionCode Name="2525C">UCVC--</LegacyFunctionCode>
+	  <LegacyEntity ID="MOVEMENT_AND_MANEUVER_AVIATION_COMPOSITE_LE" Label="WAR.GRDTRK.UNT.CBT.AVN.CMPS" Icon="Full_Frame" CloverGraphic="S-G-UCVC--_0.svg" RectangleGraphic="S-G-UCVC--_1.svg" SquareGraphic="S-G-UCVC--_2.svg" DiamondGraphic="S-G-UCVC--_3.svg">
```
Comment (abouffard) [inline instance/jmsml_D_Land_Unit.xml]: @Dbarnes1 - more FULL_FRAME fun... there are others at line 2425 and 2563.

**Comment by Dbarnes1** [comment]
@abouffard I **THINK** that this fixes all the problems. I used visual studio, as you showed me, and that greatly helped me with identifying any syntax errors that I originally typed, and it helped me with indenting correctly. I think that everyone on our team should use the JMSML visual studio soltuion file when editing in the future. Please review again.

**Comment by csmoore** [comment]
If experience has shown Visual Studio is the only way to make correct edits to this datamodel, then can we try to ensue that is reflected somewhere ( not sure where would be best - perhaps a note/warning under Requirements? https://github.com/Esri/joint-military-symbology-xml#requirements )

**Comment by BobBooth** [comment]
I think the editing SW should be left up to the reader. We could supply the JMSML vs solution file and suggest that this is useful for catching errors, but I think a text editor works too.

**Comment by abouffard** [comment]
There are a number of XML editing tools that would also flag these errors.  XMLSpy for example.  I concur that we needn't legislate what people use to edit this XML.  We just need to be sure the XML is sound before we merge it.

**Comment by csmoore** [comment]
I certainly don't want to legislate, but I would nonetheless like to capture the lessons learned from this experience (since it did seem to take longer than expected to make these seemingly straightforward edits 1+ weeks). 

Rather than text editor - can we advise to use a "tool that checks the edited XML against the schema/xsd" - perhaps that is what we are talking about here that would improve the process.

**Comment by abouffard** [comment]
Looks good Daniel, although when I tested a zip of your current JMSML repo I noticed several other XML files with problems, files you didn't include in this PR.

**Pull request merged**